site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
+- [`chrome.fileBrowserHandler`](/docs/extensions/reference/fileBrowserHandler)
+- [`chrome.fileSystemProvider`](/docs/extensions/reference/fileSystemProvider)
+- [`chrome.loginState`](/docs/extensions/reference/loginState)
+- [`chrome.printingMetrics`](/docs/extensions/reference/printingMetrics)
+- [`chrome.wallpaper`](/docs/extensions/reference/wallpaper)
+
+{% endDetails %}
+
+### Chrome 96: `declarativeNetRequestWithHostAccess` permission
+
+The new `declarativeNetRequestWithHostAccess` permission allows existing Manifest V2 extensions that
+leverage `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
+[`chrome.declarativeNetRequest`](/docs/extensions/reference/declarativeNetRequest/) API without
+requiring the end user to approve a new permissions grant.
+
+### 2021-10-27: Review violation troubleshooting updates {: #2021-10-27-reivew-troubleshooting }
 
 The [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/) page has been
 updated to provide developers with more detailed guidance for common reasons for rejection.
 
-### Manifest V2 support timeline
+### Chrome 95: Inject scripts directly into pages {: #m95-page-script-injection }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting) API's
+[`executeScript()`](/docs/extensions/reference/scripting/#method-executeScript) method can now
+inject scripts directly into a page's main world. Previously, extensions could only inject directly
+into the extension's isolated world. For more information on isolated worlds, see the documentation
+on [content scripts](/docs/extensions/mv3/content_scripts/#isolated_world).
+
+### Chrome 95: Promise support for Storage API {: #m95-storage-promise-support }
+
+Methods on the Manifest V3 version of the [`chrome.storage`](/docs/extensions/reference/storage/)
+API now return promises.
+
+### Chrome 95: Cross origin isolation opt-in now required {: #m95-coi-required }
+
+Extensions and apps must now opt into [cross origin
+isolation](https://web.dev/cross-origin-isolation-guide/) in order to use features like
+SharedArrayBuffer. Documentation on this feature is available for both [Manifest
+V2](/docs/extensions/mv2/cross-origin-isolation/) and [Manifest
+V3](/docs/extensions/mv3/cross-origin-isolation/) is available.
 
-#### Published 2021.09.23
+### 2021-09-23: Manifest V2 support timeline {: #manifest-v2-support-timeline }
 
 The Manifest V2->V3 transition timeline was [announced in this blog post](/blog/mv2-transition/) and
 a more detailed [timeline page](/docs/extensions/mv3/mv2-sunset) was published.
 
-### Two step verification enforcement
+### Chrome 94: Declarative net request static ruleset changes
 
-#### Published 2021.08.04
+The [`chrome.declarativeNetRequest`](/docs/extensions/reference/declarativeNetRequest/) API static
+ruleset implementation has been adjusted in order to give developers more flexibility. Previously
```

Comment:
```suggestion
ruleset has been adjusted to give developers more flexibility. Previously
```

Reply:
Removal of 'implementation'. Again, this may seem minor. Distinguishing the implementation from the design is only relevant to audiences who deal with both. To a user of Extensions, it's not an implementation. It's just the way it works.

Reply:
Dropping this language in favor of [@rdcronin's suggestion](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746792080).